core/src/main/java/org/kiwix/kiwixmobile/core/bookmark/BookmarksContract.kt

```diff
@@ -19,10 +19,10 @@ package org.kiwix.kiwixmobile.core.bookmark
 
 import org.kiwix.kiwixmobile.core.base.BaseContract
 
-internal interface BookmarksContract {
+interface BookmarksContract {
   interface View : BaseContract.View<Presenter> {
-    fun updateBookmarksList(bookmarks: List<BookmarkItem>)
-    fun notifyBookmarksListFiltered(bookmarks: List<BookmarkItem>)
+    fun updateBookmarksList(bookmarksList: List<BookmarkItem>)
+    fun notifyBookmarksListFiltered(bookmarksList: List<BookmarkItem>)
```

Comment:
why rename this? `bookmarks` is fine

Reply:
Lint was complaining that the name should be same where it is declared and used

Reply:
That is true but I prefer bookmarks, conveys the same amount of information as `bookmarksList` and list is already there in the type anyway if somebody gets curious